src/SFA.DAS.ApprenticeFeedback.Api/Startup.cs

```diff
@@ -79,7 +79,7 @@ public void ConfigureServices(IServiceCollection services)
                     {
                         o.Filters.Add(new AuthorizeFilter(PolicyNames.Default));
                     }
-                }).SetCompatibilityVersion(CompatibilityVersion.Version_3_0);
+                });
```

Comment:
This command is a no-op in .NET 6 -- it has no effect, therefore it has been removed